.github/workflows/lint.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
+        python-version: [ "3.10" ]
```

Comment:
ooc, why do we need whitespace here?

Reply:
ah I think this was just my formatter running on save, should be a noop (but I did intentionally change the python version to be consistently 3.10)